### PR TITLE
Clarify "What's new" instructions

### PR DESCRIPTION
**Pull request by rhetoric101**
Reviewer: 

* Please confirm these two sub-bullets that should clarify what goes into the body of the "What's new" PR.
* I had to use a real tab to indent the code snippet in step 2, but the code snippet in step three used the normal two-space fake tab. This was a workaround to fix failing indentation due to something inside the code snippet. So, the code snippets should have the same indentation (in steps 2 and 3).
* I inserted warnings about using local images only. See the comments in the image section of the first large code snippet (step 2), and then the general section on images later on in the doc. Both explain that users can't link to externally hosted images (on other sites).
* I added isFeatured information in the front matter section.

### CODE REVIEW

**Review by bradleycamacho — APPROVED** [review]
@rhetoric101 looks good to me, but I think @deven needs to review the changes to the gatsby config files!

File: src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx [hunk 1]
```diff
@@ -35,93 +35,95 @@ Here are some tips to ensure a smooth process:
...
-## Front matter tips (remove section this before publishing)
+	```md
+	---
+	title: 'Insert a title here' 
+	summary: 'Insert a brief description of the feature' 
+	releaseDate: 'yyyy-mm-dd' 
+	learnMoreLink: 'https://example.com' 
+	getStartedLink: 'https://example.com'
+	---
+	## Front matter tips (remove section this before publishing)
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx:46]: What is "(remove section this before publishing)" supposed to mean?

Comment (urbiz-nr) [reply src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx:46]: Is it "(remove this section before publishing)"?

**Review by urbiz-nr — DISMISSED** [review]
Looks good, but please check my comment.

File: src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx (hunk 1 above)
Comment (urbiz-nr) [reply src/content/docs/style-guide/writing-docs/article-templates/whats-new-template.mdx:46]: It appears a few times...

**Pull request merged**